Src/coffeescript/cql-execution/test/elm/interval/test.coffee

```diff
@@ -33,7 +33,7 @@ describe 'Interval', ->
     @closed.high.exec(@ctx).should.eql new DateTime(2013, 1, 1)
 
   it 'should exec to native Interval datatype', ->
-    ivl = @open.exec(@cql)
+    ivl = @open.exec(@ctx)
```

Comment:
Ha.  Oops.  That bug must have been written on a late night with too much CQL on the mind.